keyboards/mechlovin/zed60/chconf.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2022 QMK
```

Comment:
This file shouldnt be needed any more, as the defaults were updated last breaking changes.

Reply:
Deleted, thank you.